pyproject.toml

```diff
@@ -72,7 +73,8 @@ piccolo = ["piccolo"]
 motor = ["motor"]
 mongoengine = ["mongoengine"]
 beanie =  ["beanie"]
-all = ["sqlmodel", "SQLAlchemy", "databases", "orm", "ormar", "tortoise-orm", "asyncpg", "django", "piccolo", "motor", "mongoengine", "pony", "beanie", "sqlakeyset"]
+scylla = ["scylla-driver"]
+all = ["sqlmodel", "SQLAlchemy", "databases", "orm", "ormar", "tortoise-orm", "asyncpg", "django", "piccolo", "motor", "mongoengine", "pony", "beanie", "sqlakeyset", "scylla"]
```

Comment:
I think it should be `scylla-driver`, not `scylla`.

Reply:
changed